Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017, 2019 the Velero contributors.
```

Comment:
Copyright year should be 2020

Reply:
Completely forgot about that, thanks for the feedback. Fixed now.